### PR TITLE
Clarify instructions

### PR DESCRIPTION
**Pull request by vertige**
Misleading instructions, later clarified in class, that created a lot of confusion about Wave 1.

# Solar System
Congratulations! You're submitting your assignment.

## Comprehension Questions

|  Question 	|  Answer 	|
|---	|---	|
|  What was the purpose of the `initialize` method in your class?	|   	|
|  Describe an instance variable you used and what you used it for.	|   	|
|  Describe what the difference would be if your `SolarSystem` used an `Array` vs a `Hash`. 	|   	|
|  Do you feel like you used consistent indentation throughout your code? 	|   	|

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Let's make a planetary system!
 6.  Hoth
 ```
 - Write code to test your SolarSystem
-- Instead of Strings for planets, modify SolarSystem's `initialize` method to take a list of hashes where each planet is sent as a hash with at least 5 attributes.  
+- Instead of Strings for planets, modify `class SolarSystem` methods to take a list of hashes where each planet is sent as a hash with at least 5 attributes.
```
Comment (CheezItMan) [inline README.md:28]: I would suggest instead to `use a list of hashes` instead of take, since take implies they'll all take it as a parameter.

**Comment by vertige** [comment]
How about?:

Instead of a list of `Strings` for planets, take a list of `Hashes` where
each planet is a hash with at least 5 attributes. Modify your code as
necessary.

File: README.md (hunk 1 above)
Comment (tildeee) [reply README.md:28]: 👍 with me

**Comment by tildeee** [comment]
@vertige Hey Jessica, I'm probably going to close this PR because it's checking against your master branch at `vertige/master`, so it also happens to have your Solar System submission :P

We can talk about making feature branches at a later time so this doesn't happen in the future.

**Comment by vertige** [comment]
@tildeee, Aiyyyy! Thanks :) I've been having trouble with a few git related things it might be nice to go over if there's time.

**Pull request closed without merging**